2022-08-04 22:17:29,682 INFO    MainThread:12688 [wandb_setup.py:_flush():76] Configure stats pid to 12688
2022-08-04 22:17:29,682 INFO    MainThread:12688 [wandb_setup.py:_flush():76] Loading settings from C:\Users\rkaha\.config\wandb\settings
2022-08-04 22:17:29,682 INFO    MainThread:12688 [wandb_setup.py:_flush():76] Loading settings from C:\Users\rkaha\Documents\hopsworks\hopsworks-tutorials\nyc_taxi_fares\wandb\settings
2022-08-04 22:17:29,683 INFO    MainThread:12688 [wandb_setup.py:_flush():76] Loading settings from environment variables: {'_require_service': 'True'}
2022-08-04 22:17:29,683 INFO    MainThread:12688 [wandb_setup.py:_flush():76] Inferring run settings from compute environment: {'program': '<python with no main file>'}
2022-08-04 22:17:29,683 INFO    MainThread:12688 [wandb_setup.py:_flush():76] Applying login settings: {'api_key': '***REDACTED***'}
2022-08-04 22:17:29,685 INFO    MainThread:12688 [wandb_init.py:_log_setup():461] Logging user logs to C:\Users\rkaha\Documents\hopsworks\hopsworks-tutorials\nyc_taxi_fares\wandb\run-20220804_221729-ld19hbke\logs\debug.log
2022-08-04 22:17:29,685 INFO    MainThread:12688 [wandb_init.py:_log_setup():462] Logging internal logs to C:\Users\rkaha\Documents\hopsworks\hopsworks-tutorials\nyc_taxi_fares\wandb\run-20220804_221729-ld19hbke\logs\debug-internal.log
2022-08-04 22:17:29,686 INFO    MainThread:12688 [wandb_init.py:_jupyter_setup():411] configuring jupyter hooks <wandb.sdk.wandb_init._WandbInit object at 0x000001CFA05A4CA0>
2022-08-04 22:17:29,686 INFO    MainThread:12688 [wandb_init.py:init():495] calling init triggers
2022-08-04 22:17:29,686 INFO    MainThread:12688 [wandb_init.py:init():498] wandb.init called with sweep_config: {}
config: {}
2022-08-04 22:17:29,686 INFO    MainThread:12688 [wandb_init.py:init():514] re-initializing run, found existing run on stack: 3qhv2xw1
2022-08-04 22:17:29,690 INFO    MainThread:12688 [wandb_run.py:_finish():1727] finishing run rktraz_/nyc_taxi_fares/3qhv2xw1
2022-08-04 22:17:29,690 INFO    MainThread:12688 [wandb_run.py:_atexit_cleanup():1930] got exitcode: 0
2022-08-04 22:17:29,691 INFO    MainThread:12688 [wandb_run.py:_restore():1913] restore
2022-08-04 22:17:29,691 INFO    MainThread:12688 [wandb_run.py:_restore():1919] restore done
2022-08-04 22:17:29,699 INFO    MainThread:12688 [wandb_run.py:_on_finish():2220] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 779
  total_bytes: 779
}

2022-08-04 22:17:30,038 INFO    MainThread:12688 [wandb_run.py:_on_finish():2220] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 779
  total_bytes: 779
}

2022-08-04 22:17:30,722 INFO    MainThread:12688 [wandb_run.py:_on_finish():2220] got exit ret: file_counts {
  wandb_count: 4
}
pusher_stats {
  uploaded_bytes: 779
  total_bytes: 4470
}

2022-08-04 22:17:30,838 INFO    MainThread:12688 [wandb_run.py:_on_finish():2220] got exit ret: file_counts {
  wandb_count: 4
}
pusher_stats {
  uploaded_bytes: 779
  total_bytes: 4470
}

2022-08-04 22:17:30,950 INFO    MainThread:12688 [wandb_run.py:_on_finish():2220] got exit ret: file_counts {
  wandb_count: 4
}
pusher_stats {
  uploaded_bytes: 779
  total_bytes: 4470
}

2022-08-04 22:17:31,055 INFO    MainThread:12688 [wandb_run.py:_on_finish():2220] got exit ret: file_counts {
  wandb_count: 4
}
pusher_stats {
  uploaded_bytes: 4470
  total_bytes: 4470
}

2022-08-04 22:17:31,162 INFO    MainThread:12688 [wandb_run.py:_on_finish():2220] got exit ret: file_counts {
  wandb_count: 4
}
pusher_stats {
  uploaded_bytes: 4470
  total_bytes: 4470
}

2022-08-04 22:17:31,272 INFO    MainThread:12688 [wandb_run.py:_on_finish():2220] got exit ret: file_counts {
  wandb_count: 4
}
pusher_stats {
  uploaded_bytes: 4470
  total_bytes: 4470
}

2022-08-04 22:17:31,378 INFO    MainThread:12688 [wandb_run.py:_on_finish():2220] got exit ret: file_counts {
  wandb_count: 4
}
pusher_stats {
  uploaded_bytes: 4470
  total_bytes: 4470
}

2022-08-04 22:17:31,487 INFO    MainThread:12688 [wandb_run.py:_on_finish():2220] got exit ret: file_counts {
  wandb_count: 4
}
pusher_stats {
  uploaded_bytes: 4470
  total_bytes: 4470
}

2022-08-04 22:17:31,603 INFO    MainThread:12688 [wandb_run.py:_on_finish():2220] got exit ret: file_counts {
  wandb_count: 4
}
pusher_stats {
  uploaded_bytes: 4470
  total_bytes: 4470
}

2022-08-04 22:17:32,017 INFO    MainThread:12688 [wandb_run.py:_on_finish():2220] got exit ret: file_counts {
  wandb_count: 4
}
pusher_stats {
  uploaded_bytes: 4470
  total_bytes: 4470
}

2022-08-04 22:17:32,369 INFO    MainThread:12688 [wandb_run.py:_on_finish():2220] got exit ret: done: true
exit_result {
}
file_counts {
  wandb_count: 4
}
pusher_stats {
  uploaded_bytes: 4470
  total_bytes: 4470
}
local_info {
}

2022-08-04 22:17:33,514 INFO    MainThread:12688 [wandb_run.py:_footer_history_summary_info():3304] rendering history
2022-08-04 22:17:33,514 INFO    MainThread:12688 [wandb_run.py:_footer_history_summary_info():3336] rendering summary
2022-08-04 22:17:33,519 INFO    MainThread:12688 [wandb_run.py:_footer_sync_info():3260] logging synced files
2022-08-04 22:17:33,529 INFO    MainThread:12688 [wandb_init.py:init():548] starting backend
2022-08-04 22:17:33,530 INFO    MainThread:12688 [wandb_init.py:init():552] setting up manager
2022-08-04 22:17:33,537 INFO    MainThread:12688 [backend.py:_multiprocessing_setup():97] multiprocessing start_methods=spawn, using: spawn
2022-08-04 22:17:33,541 INFO    MainThread:12688 [wandb_init.py:init():558] backend started and connected
2022-08-04 22:17:33,574 INFO    MainThread:12688 [wandb_run.py:_label_probe_notebook():1105] probe notebook
2022-08-04 22:17:33,576 INFO    MainThread:12688 [wandb_run.py:_label_probe_notebook():1115] Unable to probe notebook: 'NoneType' object has no attribute 'get'
2022-08-04 22:17:33,576 INFO    MainThread:12688 [wandb_init.py:init():636] updated telemetry
2022-08-04 22:17:33,654 INFO    MainThread:12688 [wandb_init.py:init():667] communicating run to backend with 30 second timeout
2022-08-04 22:17:35,062 INFO    MainThread:12688 [wandb_run.py:_on_init():1981] communicating current version
2022-08-04 22:17:35,274 INFO    MainThread:12688 [wandb_run.py:_on_init():1985] got version response 
2022-08-04 22:17:35,275 INFO    MainThread:12688 [wandb_init.py:init():700] starting run threads in backend
2022-08-04 22:17:35,476 INFO    MainThread:12688 [wandb_run.py:_console_start():1961] atexit reg
2022-08-04 22:17:35,477 INFO    MainThread:12688 [wandb_run.py:_redirect():1819] redirect: SettingsConsole.WRAP_RAW
2022-08-04 22:17:35,477 INFO    MainThread:12688 [wandb_run.py:_redirect():1884] Wrapping output streams.
2022-08-04 22:17:35,477 INFO    MainThread:12688 [wandb_run.py:_redirect():1906] Redirects installed.
2022-08-04 22:17:35,481 INFO    MainThread:12688 [wandb_init.py:init():732] run started, returning control to user process
2022-08-04 22:17:37,708 INFO    MainThread:12688 [wandb_init.py:_pause_backend():374] pausing backend
2022-08-04 22:17:37,708 INFO    MainThread:12688 [jupyter.py:save_ipynb():377] not saving jupyter notebook
2022-08-04 22:26:14,189 INFO    MainThread:12688 [wandb_init.py:_resume_backend():383] resuming backend
2022-08-04 22:26:14,193 INFO    MainThread:12688 [wandb_init.py:_pause_backend():374] pausing backend
2022-08-04 22:26:14,193 INFO    MainThread:12688 [jupyter.py:save_ipynb():377] not saving jupyter notebook
2022-08-04 22:26:19,778 INFO    MainThread:12688 [wandb_init.py:_resume_backend():383] resuming backend
2022-08-04 22:26:19,781 INFO    MainThread:12688 [wandb_init.py:_pause_backend():374] pausing backend
2022-08-04 22:26:19,781 INFO    MainThread:12688 [jupyter.py:save_ipynb():377] not saving jupyter notebook
2022-08-04 22:26:21,605 INFO    MainThread:12688 [wandb_init.py:_resume_backend():383] resuming backend
2022-08-04 22:26:22,553 INFO    MainThread:12688 [wandb_init.py:_pause_backend():374] pausing backend
2022-08-04 22:26:22,554 INFO    MainThread:12688 [jupyter.py:save_ipynb():377] not saving jupyter notebook
2022-08-04 22:26:24,221 INFO    MainThread:12688 [wandb_init.py:_resume_backend():383] resuming backend
2022-08-04 22:26:24,226 INFO    MainThread:12688 [pyagent.py:run():316] Starting sweep agent: entity=None, project=None, count=5
2022-08-04 22:26:32,037 WARNING MsgRouterThr:12688 [router.py:message_loop():76] message_loop has been closed
2022-08-04 22:26:51,162 INFO    MainThread:12688 [jupyter.py:save_ipynb():377] not saving jupyter notebook
